### PR TITLE
feat:Remove inline javascript code from Cloudbees workflow

### PR DESCRIPTION
**Pull request by NeetigyaPod**
### Testing done
Maven tests pass and mvn hpi:run does not reveal time zone anymore.

```[tasklist]
### Submitter checklist
- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue
```
Converted inline script to recommended approach as mentioned in https://www.jenkins.io/doc/developer/security/csp/#inline-javascript-blocks .
Issue: https://issues.jenkins.io/browse/JENKINS-72137

### CODE REVIEW

**Review by Kevin-CB — COMMENTED** [review]
I tested your PR locally, and it works as expected.
I also confirmed with the CSP plugin that no violations were present anymore.

For the CI errors, I don't think it's related to your changes

**Comment by NeetigyaPod** [comment]
Oh thank you! Have accepted the changes, sorry for the extra spaces.

**Comment by NeetigyaPod** [comment]
Hi @Kevin-CB can we merge this before end of October? I would like it to be a part of my hacktoberfest contribution.

**Review by Kevin-CB — APPROVED** [review]
Hi @NeetigyaPod, even though I approved this PR, I can't merge it as I'm not a maintainer of this plugin.

**Comment by NeetigyaPod** [comment]
@olamy can you please help merge this PR by end of October? I would like it to be a part of my hacktoberfest contribution. Would appreciate a lot.

**Pull request merged**
